src/rpm_ostree/cli_deploy.rs

```diff
@@ -99,11 +99,21 @@ fn invoke_cli_deploy(release: Release, allow_downgrade: bool) -> Result<Release>
     fail_point!("deploy_locked_ok", |_| Ok(release.clone()));
 
     let mut cmd = std::process::Command::new("rpm-ostree");
-    cmd.arg("deploy")
-        .arg("--lock-finalization")
-        .arg("--skip-branch-check")
-        .arg(format!("revision={}", release.checksum))
-        .env("RPMOSTREE_CLIENT_ID", "zincati");
+    if release.is_oci {
+        // TODO use --custom-origin-url and --custom-origin-description
+        cmd.arg("rebase")
+            .arg(format!(
+                "ostree-remote-image:fedora:docker://{}",
```

Comment:
Here I hard-coded the fedora ostree remote reference to get the ostree signature verification. 
In the future, we will sign the OCI image directly and switch to bootc, making this irrelevant. 
I suppose this will break Zincati for OCI images not created by coreos. I need to test with a layered build to see if the original ostree signature is still verified or if it breaks.